src/app/pages/resources/api/chpl-api.scss

```diff
@@ -4,26 +4,82 @@
         color: #333;
     }
 
-    .endpoint {
+    .btn.authorize{
+         color:#66926d;
+         border-color: #66926d;
+    }
+
+    .btn.authorize svg{
+        fill:#66926d;
+   }
+
+   .info{
+    border-top: 1px solid rgba(59, 65, 81, 0.3);
+    padding-top: 32px;
+   }
+
+   .info{
+    a {
+        color: #156dac;
+        text-decoration: underline;
+      }
+    }
+
+   .info .title small.version-stamp{
+       background-color: #384903;
+   }
+
+   .info .title {
+    color: #3b4151;
+    font-size: 1.75em;
+    margin: 0;
+    font-weight: 400;
+    font-family: 'Lato';
+   }
+
+   .opblock.opblock-get .opblock-summary{
+       border-color:#156dac;
+   }
 
-        h4 a,
-        a {
-            color: #337ab7;
-        }
+   .opblock.opblock-get .opblock-summary-method{
+       background:#156dac;
+   }
 
-        .endpoint-actions a,
-        .endpoint-heading span {
-            color: #333;
-        }
+   .opblock.opblock-put .opblock-summary{
+    border-color:#cc8400;
     }
 
-    .operation {
+    .opblock.opblock-put .opblock-summary-method{
+     background:#cc8400;
+    }
+
+   .opblock.opblock-post .opblock-summary{
+       border-color:#66926d;
+   }
+
+   .opblock.opblock-post .opblock-summary-method{
+    background:#66926d;
+   }
+
+   .opblock.opblock-delete .opblock-summary{
+    border-color:#c44f65;
+    }
+
+    .opblock.opblock-delete .opblock-summary-method{
+    background:#c44f65;
+    }
 
-        .content {
+   pre{
```

Comment:
This is a global change to an element, and so should either be in the themes.js file, or the index.scss, or if we can scope it to just the `pre` tags in this file, put in wherever the actual swagger styling things go